wagtail/admin/templatetags/wagtailadmin_tags.py

```diff
@@ -771,6 +770,49 @@ def icon(name=None, classname=None, title=None, wrapped=False, class_name=None):
             category=RemovedInWagtail60Warning,
         )
 
+    deprecated_icons = [
+        "angle-double-left",
+        "angle-double-right",
+        "arrow-down-big",
+        "arrow-up-big",
+        "arrows-up-down",
+        "chain-broken",
+        "dots-vertical",
+        "ellipsis-v",
+        "horizontalrule",
+        "repeat",
+        "reset",
+        "undo",
+        "wagtail-inverse",
+    ]
```

Comment:
All of those weren’t used anywhere in Wagtail core _before_ I made my changes, except for `arrow-down-big`, `arrow-up-big`, and `ellipsis-v`.